tests/e2e/tests/01-enable-proxy-mode/00-scale-target-down-to-zero.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl scale deployment target-deployment -n default --replicas=0
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)